modules/mesos/pom.xml

```diff
@@ -36,8 +37,10 @@
 
     <properties>
         <mesos.version>0.22.0</mesos.version>
-        <ignite.latest.url>http://ignite.run/download_ignite.php</ignite.latest.url>
-        <ignite.direct.url>https://archive.apache.org/dist/ignite/%s/apache-ignite-fabric-%s-bin.zip</ignite.direct.url>
+        <ignite.archive.url>https://archive.apache.org/dist/ignite/%s/apache-ignite-fabric-%s-bin.zip</ignite.archive.url>
+        <ignite.version.url>https://ignite.apache.org/latest</ignite.version.url>
```

Comment:
Should use build version and not the latest one. Don't know what to do for snapshot builds.

Reply:
@alamar Sorry, I don't quite understand your concern. https://ignite.apache.org/latest will not give you the snapshot or nightly builds, just the latest stable one, AFAIK.

Can you please explain?

Reply:
@alamar Well, ignite-mesos 2.x has no dependencies on Ignite, therefore it doesn't care what version to run -- nothing will be broken. It's just a driver, and the user decides what version to run.
Looking at ignite-mesos 2.x may be misleading ;)